ml-agents/mlagents/trainers/rl_trainer.py

```diff
@@ -1,25 +1,37 @@
 # # Unity ML-Agents Toolkit
 import logging
-from typing import Dict
+from typing import Dict, List, Any
 from collections import defaultdict
+import abc
+import time
 
+from mlagents.tf_utils import tf
 from mlagents.trainers.tf_policy import TFPolicy
+from mlagents_envs.timers import set_gauge
 from mlagents.trainers.buffer import AgentBuffer
+from mlagents.trainers.trajectory import Trajectory
 from mlagents.trainers.trainer import Trainer, UnityTrainerException
+from mlagents.trainers.agent_processor import AgentManagerQueue
 from mlagents.trainers.components.reward_signals import RewardSignalResult
+from mlagents_envs.timers import hierarchical_timer
 
 LOGGER = logging.getLogger("mlagents.trainers")
 
 RewardSignalResults = Dict[str, RewardSignalResult]
 
 
-class RLTrainer(Trainer):  # pylint: disable=abstract-method
+class RLTrainer(Trainer, abc.ABC):  # pylint: disable=abstract-method
```

Comment:
why do you need to inherit from abc.ABC? linter complaints or runtime errors?

Reply:
Oh, I missed the new abstract methods below. But do you still need this, since Trainer already inherits from ABC?

Reply:
I suppose it's also to prevent the instantiation of RLTrainer class. But perhaps it isn't necessary